### PR TITLE
Add OS and arch in release tarball.

### PR DESCRIPTION
**Pull request by Random-Liu**
This PR:
1) Add OS and arch in release tarball name.
2) Generate sha1sum for release tarball.
3) Update version to 1.0.0-beta.0 to prepare for the release.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: hack/install-deps.sh [hunk 1]
```diff
@@ -146,5 +146,6 @@ ${sudo} bash -c 'cat >'${CRICTL_CONFIG_DIR}'/crictl.yaml <<EOF
 runtime-endpoint: /var/run/cri-containerd.sock
 EOF'
 
-# Clean the tmp GOPATH dir
-${sudo} rm -r ${GOPATH}
+# Clean the tmp GOPATH dir. Use sudo because runc build generates
+# some privileged files.
+sudo rm -r ${GOPATH}
```
Comment (mikebrow) [inline hack/install-deps.sh]: did you mean to remove {} from here and above?

Comment (Random-Liu) [reply hack/install-deps.sh]: Added `${sudo}` back, use `-rf` instead.

File: hack/install-deps.sh [hunk 2]
```diff
@@ -146,5 +146,6 @@ ${sudo} bash -c 'cat >'${CRICTL_CONFIG_DIR}'/crictl.yaml <<EOF
 runtime-endpoint: /var/run/cri-containerd.sock
 EOF'
 
-# Clean the tmp GOPATH dir
-${sudo} rm -r ${GOPATH}
+# Clean the tmp GOPATH dir. Use sudo because runc build generates
+# some privileged files.
+${sudo} rm -rf ${GOPATH}
```
Comment (mikebrow) [inline hack/install-deps.sh]: scary, but better

Comment (Random-Liu) [reply hack/install-deps.sh]: Is this better?

Comment (mikebrow) [reply hack/install-deps.sh]: Much!

**Pull request merged**